paasta_tools/kubernetes_tools.py

```diff
@@ -1485,7 +1486,11 @@ def get_kubernetes_containers(
                 )
             ),
             name=self.get_sanitised_instance_name(),
-            liveness_probe=self.get_liveness_probe(service_namespace_config),
+            liveness_probe=(
+                self.get_liveness_probe(service_namespace_config)
+                if include_liveness_probe
+                else None
+            ),
             readiness_probe=self.get_readiness_probe(service_namespace_config),
```

Comment:
we probably also don't want to include the readiness probe, right?

Reply:
I have some pretty incredible tunnel vision. But yeah, probably.